drake/doc/developers.rst

```diff
@@ -139,10 +139,28 @@ Continuous Integration Notes
     CDash <https://drake-cdash.csail.mit.edu/index.php?project=Drake>
     jenkins
 
-IDE Notes
-=========
-* `CLion <https://github.com/tkoolen/drake/wiki/CLion-setup-(experimental)>`_
-* `Eclipse <https://github.com/tkoolen/drake/wiki/Eclipse-setup-(experimental)>`_
+IDE and Text Editor Notes
+=========================
+
+.. toctree::
+    :maxdepth: 1
+
+    clion
+    `Eclipse <https://github.com/tkoolen/drake/wiki/Eclipse-setup-(experimental)>`_
+
+
+Sublime Text
+------------
+
+Recommended packages to install:
+
+1. https://packagecontrol.io/packages/SublimeLinter-cpplint
+
+To display the current file's full path in the title bar, open your user preferences by going to "Sublime Text," "Preferences," "Settings - User." Then add the following to your user preferences:
```

Comment:
We should add a note here saying this is only needed on OS X since Sublime inexplicably by default shows the full path on Windows and Linux.